code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -265,8 +271,8 @@
 /////////////
 
 /obj/item/weapon/storage/fancy/cigarettes/cigars
-	name = "\improper premium cigar case"
-	desc = "A case of premium cigars. Very expensive."
+	name = "\improper cigar case"
+	desc = "A case of cheap cigars. Still, very expensive."

```

Comment:
Might want to use cheaper rather than cheap as it looks weird calling cheap cigars expensive